### PR TITLE
configure hatchery for nextflow workspaces

### PR DESCRIPTION
**Pull request by jmontmaxwell**
Link to Jira ticket if there is one:

### Environments
BRH Staging

### Description of changes

- Update hatchery config to remove old nextflow workspace and add the two enxtflow workspace from QA BRH
- Update manifest.json config for hatchery 2.0.1

### CODE REVIEW

File: brhstaging.data-commons.org/manifests/hatchery/hatchery.json [hunk 1]
```diff
@@ -106,14 +106,14 @@
       "target-port": 8888,
       "cpu-limit": "4.0",
       "memory-limit": "10Gi",
-      "name": "(Nextflow) Test Nextflow in BRH",
-      "image": "quay.io/cdis/jupyter-notebook:feat_nextflow",
+      "name": "(Beta) Nextflow with CPU instances",
+      "image": "quay.io/cdis/jupyter-notebook:nextflow-feat_nextflow-welcome",
       "env": {
-        "FRAME_ANCESTORS": "https://brhstaging.data-commons.org"
+        "FRAME_ANCESTORS": "https://qa-brh.planx-pla.net"
```
Comment (aartivnkt) [inline brhstaging.data-commons.org/manifests/hatchery/hatchery.json:112]: this one is still `brhstaging` right?

File: brhstaging.data-commons.org/manifests/hatchery/hatchery.json [hunk 2]
```diff
@@ -125,15 +125,96 @@
...
+      }
+    },
+    {
+      "target-port": 8888,
+      "cpu-limit": "4.0",
+      "memory-limit": "10Gi",
+      "name": "(Beta) Nextflow with GPU instances",
+      "image": "quay.io/cdis/jupyter-notebook:nextflow-feat_nextflow-welcome",
+      "env": {
+        "FRAME_ANCESTORS": "https://qa-brh.planx-pla.net"
```
Comment (aartivnkt) [inline brhstaging.data-commons.org/manifests/hatchery/hatchery.json:169]: same with above, change to `brhstaging`

**Review by aartivnkt — CHANGES_REQUESTED** [review]
left a couple of comments

File: brhstaging.data-commons.org/manifests/hatchery/hatchery.json (hunk 1 above)
Comment (jmontmaxwell) [reply brhstaging.data-commons.org/manifests/hatchery/hatchery.json]: fixed

**Review by aartivnkt — DISMISSED** [review]

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**